src/GraphQL.ApiTests/net6/GraphQL.approved.txt

```diff
@@ -1556,15 +1557,6 @@ namespace GraphQL.Resolvers
         public System.Threading.Tasks.ValueTask<object?> ResolveAsync(GraphQL.IResolveFieldContext context) { }
     }
 }
-namespace GraphQL.Subscription
```

Comment:
So this namespace is gone! I suggest to move `ObservableExtensions` into `GraphQL` namespace.